ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/operations/SelfDestructOperation.java

```diff
@@ -70,8 +70,6 @@ public void execute(final MessageFrame frame) {
       final MessageFrame frame,
       final EnumSet<ExceptionalHaltReason> previousReasons,
       final EVM evm) {
-    return frame.isStatic()
-        ? Optional.of(ExceptionalHaltReason.ILLEGAL_STATE_CHANGE)
-        : Optional.empty();
+    return Optional.of(ExceptionalHaltReason.ILLEGAL_STATE_CHANGE).filter(__ -> frame.isStatic());
```

Comment:
Can we change it back to the ternary operator? It is easier to read! :)

Reply:
we were told filters are easier to read :dagger: :imp: 

I agree the ternary is better!